PiBridgeMaster.c

```diff
@@ -67,7 +67,8 @@ void PiBridgeMaster_Stop(void)
 {
 	my_rt_mutex_lock(&piCore_g.lockBridgeState);
 	if (piDev_g.machine_type != REVPI_CONNECT_SE &&
-	    piDev_g.machine_type != REVPI_CORE_SE)
+	    piDev_g.machine_type != REVPI_CORE_SE &&
+	    piDev_g.machine_type != REVPI_CONNECT_4)
```

Comment:
This is getting cumbersome to maintain. It would be good to just add an `unsigned int revpi_gate_supported:1;` to <a href="https://github.com/RevolutionPi/piControl/blob/1985f74eba9ebedaa310170945bc0c2e712ad7a7/piControlMain.h#L65">struct spiControlDev</a>, set it to true in <a href="https://github.com/RevolutionPi/piControl/blob/1985f74eba9ebedaa310170945bc0c2e712ad7a7/piControlMain.c#L219">piControlInit()</a> and simply check the flag in all those if-clauses.

Reply:
See https://github.com/RevolutionPi/piControl/pull/94/commits/e0ed45caf7bf42b8b0b4666838440a674901bc7c